nvi-commons/src/test/java/no/sikt/nva/nvi/common/service/CandidateTest.java

```diff
@@ -215,6 +214,7 @@ void shouldNotUpdateCandidateCreatedDateOnUpdates() {
 
     @Test
     void shouldNotUpdateReportedCandidate() {
+        // FIXME: Fix or remove this test? It doesn't do anything now.
```

Comment:
What do you think @monaullah ?

Reply:
This is an important test, I wonder why it's not completed? I can create a jira task for following this up.

Reply:
So, do not remove